azure-pipelines.yml

```diff
@@ -16,9 +18,15 @@ steps:
       versionSpec: '10.x'
     displayName: 'Install Node.js'
 
+  - task: CacheBeta@0
+    inputs:
+      key: yarn | $(Agent.OS) | yarn.lock
+      path: $(YARN_CACHE_FOLDER)
+    displayName: Cache Yarn packages
+
   - script: |
       yarn
-    displayName: yarn
+    displayName: yarn --frozen-lockfile
```

Comment:
Should you do this in the actual command too?

Reply:
okay !!